jobs/JEVS_ANALYSES_PLOTS

```diff
@@ -56,10 +57,10 @@ export RUN=${RUN:-atmos}
 
 echo $COMPATH
 
-export COMIN=${COMIN:-$(compath.py ${envir}/com/evs/${evs_ver}/)}
-export COMINanl=${COMINanl:-$(compath.py ${envir}/com/evs/${evs_ver}/stats/${COMPONENT})}
-export COMOUT=${COMOUT:-${COMIN}/${STEP}/${COMPONENT}}
-export COMOUTplots=${COMOUTplots:-${COMOUT}/${RUN}.${VDATE}}
+export COMIN=${COMIN:-$(compath.py ${envir}/com/evs/${evs_ver})}
+export COMINanl=${COMINanl:-${COMIN}/stats/${COMPONENT}}
+export COMOUT=${COMOUT:-$(compath.py -o evs/${evs_ver})}
```

Comment:
Thanks! Should we make this one use ${NET} instead of evs?
`export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}`

Reply:
Done.